script.reset_watchedstate/lib/reset_watchedstate.py

```diff
@@ -0,0 +1,114 @@
+import sys
```

Comment:
Please remove pyc files from the submission

Reply:
done